src/components/resultsbox/calculator.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { DrugList } from "../druglist";
+import GridTable from "./gridtable";
+
+const determineMinDose = (weightInput, minDosePerKg, maxDailyDose, freq) => {
+  const minDosePerDay =
+    weightInput * minDosePerKg >= maxDailyDose
+      ? maxDailyDose
+      : weightInput * minDosePerKg;
+  const minDose = divideDoseByFreq(freq, minDosePerDay);
+  return minDose;
+};
+
+const determineMaxDose = (weightInput, maxDosePerKg, maxDailyDose, freq) => {
+  const maxDosePerDay =
+    weightInput * maxDosePerKg >= maxDailyDose
+      ? maxDailyDose
+      : weightInput * maxDosePerKg;
+  const maxDose = divideDoseByFreq(freq, maxDosePerDay);
+  return maxDose;
+};
+
+const divideDoseByFreq = (freq, dose) => {
+  let dividedDose = 0;
+  if (freq === "BD") {
+    dividedDose = (dose / 2).toFixed(2);
+  } else if (freq === "TDS") {
+    dividedDose = (dose / 3).toFixed(2);
+  } else if (freq === "QDS") {
+    dividedDose = (dose / 4).toFixed(2);
+  }
+  return dividedDose;
+};
+
+const convertToSyrup = (strength, mg) => {
+  // = 1 divide by mgPerMl * mg
+  let convertToMl = ((1 / strength) * mg).toFixed(2);
+  return convertToMl;
```

Comment:
```suggestion
  return ((1 / strength) * mg).toFixed(2);
```

No use for the variable, also `let` is not correct here, should be `const`. If you used the variable for informational purposes, we can just rename the function to `convertMgToMl` I think